File: php-app/app/Models/Komponist.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Komponist extends Model
{
    use HasFactory;

    protected $table = 'komponisten';

    protected $fillable = [
        'vorname',
        'name',
        'born',
        'note',
    ];

    protected $casts = [
        'born' => 'date',
    ];

    public function werke()
    {
        return $this->hasMany(Werk::class);
    }
}

File: Data/Komponist.cs
using System.ComponentModel.DataAnnotations;

namespace MaestroNotes.Data
{
    public class Komponist
    {
        [Key]
        public int Id { get; set; }
        [MaxLength(50)]
        public string Vorname { get; set; } = "";

        [MaxLength(50)]
        public string Name { get; set; } = "";
        public DateTime? Born { get; set; }
        [MaxLength(1000)]
        public string Note { get; set; } = "";

        public DateTime? Died { get; set; }
        // ALTER TABLE Komponisten ADD Died DATETIME NULL;
    }
}
